services/src/unit-test/java/uk/gov/moj/sdt/services/BulkSubmissionServiceTest.java

```diff
@@ -188,11 +166,122 @@ void testSaveBulkSubmission() throws IOException {
         bulkSubmissionService.saveBulkSubmission(bulkSubmission);
 
         // Verify the Mock
-        verify(mockGenericDao, times(1)).persist(bulkSubmission);
+        verify(mockGenericDao).persist(bulkSubmission);
+
+        assertTrue(true, EXPECTED_TO_PASS);
+    }
+
+    @Test
+    void testSaveBulkSubmissionWithDebug() throws IOException {
+        SdtContext.getContext().setRawInXml(Utilities.getRawXml(PATH_TEST_RESOURCES, TEST_XML_VALID_2));
+        ITargetApplication targetApplication = new TargetApplication();
```

Comment:
Move this declaration to just before it's used on line 210.

Reply:
Done